Calinga.NET/Calinga.NET.csproj

```diff
@@ -6,7 +6,7 @@
     <Nullable>enable</Nullable>
     <PackageId>Calinga.NET</PackageId>
     <Description>Library to integrate Calinga in .NET projects</Description>
-    <Version>1.6.0</Version>
+    <Version>1.6.1</Version>
```

Comment:
Please update to 1.7.0 instead: Although the code has hardly been changed I want to make it clear, that we now support a new feature (custom cache services). That's why I think a minor version update is justified